cdap-ui/app/cdap/components/FieldLevelLineage/v2/Context/FllContext.tsx

```diff
@@ -45,40 +52,52 @@ export interface IContextState {
   causeSets: ITableFields;
   impactSets: ITableFields;
   showingOneField: boolean;
+  start: number;
```

Comment:
Looks like this would cause a problem in strict type checking mode (but not regular type-checking mode). start and end can be numbers or null, so I've updated the interface to reflect this.